lua/output_panel/init.lua

```diff
@@ -40,6 +40,7 @@ function M.panel()
 
   vim.api.nvim_win_set_buf(0, output_panel_bufnr)
   vim.api.nvim_win_set_height(0, 30)
+  vim.opt_local.filetype = "OutputPanel"
```

Comment:
Also can you change it to vim.bo.filetype? If that is the one that works for filetype.